### PR TITLE
Redact auth in debug tripper

### PR DESCRIPTION
**Pull request by loganintech**

### CODE REVIEW

File: pkg/uhttp/transport.go [hunk 1]
```diff
@@ -100,8 +101,16 @@ func (uat *debugTripper) RoundTrip(req *http.Request) (*http.Response, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	reg, err := regexp.Compile("Authorization: Bearer .*\n")
```
Comment (pquerna) [inline pkg/uhttp/transport.go:105]: yeah dont compile inline func, but sure, hack?

Comment (loganintech) [reply pkg/uhttp/transport.go]: I put it in an init block. "hack" or not, it's a good way to remove bearer tokens from the debug output

File: pkg/uhttp/transport.go [hunk 2]
```diff
@@ -16,6 +17,12 @@ import (
 	"golang.org/x/oauth2"
 )
 
+var authorizationRegexFilter *regexp.Regexp
```
Comment (jirwin) [inline pkg/uhttp/transport.go:20]: `var authorizationRegexFilter = regexp.MustCompile()`

Comment (loganintech) [reply pkg/uhttp/transport.go:20]: Going from compile to must compile 👍🏼

**Review by pquerna — APPROVED** [review]

**Pull request merged**